src/dbnode/storage/shard_test.go

```diff
@@ -1932,16 +1888,18 @@ func TestShardAggregateTiles(t *testing.T) {
 			BlockStart:          opts.Start,
 			BlockSize:           targetBlockSize,
 			VolumeIndex:         1,
-			PlannedRecordsCount: 2,
+			PlannedRecordsCount: 3,
 		}),
-		writer.EXPECT().WriteAll(id1, tags1, write1, gomock.Any()),
-		writer.EXPECT().WriteAll(id2, tags2, write2, gomock.Any()),
-		writer.EXPECT().WriteAll(id3, tags3, write3, gomock.Any()),
 		writer.EXPECT().Close(),
 	)
 
+	targetNs := NewMockNamespace(ctrl)
+	aggregator.EXPECT().
+		AggregateTiles(opts, targetNs, sourceShard.ID(), gomock.Len(2), writer).
+		Return(int64(4), nil)
```

Comment:
nit: can this 4 be a named variable to give it some context?

Reply:
👍